cypress/e2e/CMS/$MiniLandingPageTest/index.js

```diff
@@ -232,4 +232,32 @@ Then('the promo image is matching the earlier selected image', () => {
         const actSrc = source.replace(/\?itok=[\S]+/, '').replace(/^(.*?)\/public/, '')
         expect(actSrc).to.include(extractedImageName.replaceAll('_', '-').replace('article', ''))
     })
-});
\ No newline at end of file
+});
+             /* ----- Translation of Mini Landinga ----- */
+And('banner image has requirements translated as follows', (dataTable) => {
+    for (const { text } of dataTable.hashes()) {
+        cy.get("div[class*='ld-banner-image-0']").should('include.text', text)
+    }
+})
+
+And('the following content sections are displayed', (dataTable) => {
+    for (const { section } of dataTable.hashes()) {
+        cy.get("div[class*='paragraph-type-ti']").should('include.text', section)
+    }
+})
+
+And('Dynamic List has options translated as {string}', (TranslatedOption) => {
+    cy.get(`tbody summary[role='button'] span:contains("${TranslatedOption}")`).should('be.visible')
+})
+
+And('button to add content was translated to start with {string}', (TranslatedOption) => {
+    cy.get("input[value='Añadir Content Block']").contains(TranslatedOption)
```

Comment:
Anadir should not be hardcoded , also I do not see a assertion here 

cy.get("input[value*='Content Block']").should("include.text",TranslatedOption)